src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
-export interface Figure {
+function Rounding(value: number): number {
+  return Math.floor(value * 100) / 100;
+}
 
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
 }
 
-export class Triangle {
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape: Shape = Shape.Triangle;
+
+  constructor(
+    public color: Color,
+    public SideA: number,
+    public SideB: number,
+    public SideC: number,
+  ) {
+    if (this.SideA <= 0 || this.SideB <= 0 || this.SideC <= 0) {
+      throw new Error('Side length can`t be 0');
+    }
```

Comment:
```suggestion
    if (this.SideA <= 0 || this.SideB <= 0 || this.SideC <= 0) {
      throw new Error('Side length should be positive);
    }
```
You have check not only for 0 here